ecs.tf

```diff
@@ -78,6 +78,44 @@ module "app_container_definition" {
   }
 }
 
+module "datadog_container_definition" {
+  source                   = "git::https://github.com/cloudposse/terraform-aws-ecs-container-definition.git?ref=tags/0.21.0"
+  container_name           = "datadog-agent"
+  container_image          = "datadog/agent:latest"
+  container_cpu            = "10"
+  container_memory         = "128"
+  essential                = false
+  readonly_root_filesystem = false
+  environment = [
+    {
+      name  = "ECS_FARGATE",
+      value = true
+    },
+    {
+      name  = "DD_APM_ENABLED",
+      value = true
+    }
+  ]
+  secrets = [
+    {
+      name: "DD_API_KEY",
+      valueFrom: var.datadog_api_key_from
```

Comment:
I've seen this being named as `value_from` in the `ecs-params.yml` file, but I'm assuming that you're not using that method and this goes straight to the task definition json file?

Reply:
I got it from the docs for the image source we're using 

https://github.com/cloudposse/terraform-aws-ecs-container-definition/blob/eee8fbe907b65becbcf338e930d365e71fe3b91d/variables.tf#L93

I think it gets passed straight